templates/industrial/index.html

```diff
@@ -0,0 +1,695 @@
+      <!-- rtp section left end -->
+    </div>
+    <div class="col-4 p-divider__block">
+      <!-- rtp section right start -->
+      <div class='p-heading-icon--muted'>
+        <div class='p-heading-icon__header'>
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/f6d6f78b-Case+study.svg",
+              alt="",
+              height="28",
+              width="32",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
+            ) | safe
+          }}
+          <h4 class='p-heading-icon__title'>Case study</h4>
+        </div>
+      </div>
+      <p>Ubuntu Core makes the Dell Edge Gateways ultra reliable in challenging industrial settings with transactional software updates.</p>
+      <p><a href='https://ubuntu.com/blog/the-dell-edge-gateway-3000-launches-with-ubuntu-core-16' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'product page', 'eventAction' : 'core - left', 'eventLabel' : 'Case study - IMS Evolve ', 'eventValue' : '1' });'>Dell gateway launches with Ubuntu&nbsp;&rsaquo;</a></p>
+      <!-- rtp section right end -->
+    </div>
+    <div class="col-4 p-divider__block">
+      <!-- rtp section right start -->
+      <div class='p-heading-icon--muted'>
+        <div class='p-heading-icon__header'>
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/f6d6f78b-Case+study.svg",
+              alt="",
+              height="28",
+              width="32",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
+            ) | safe
+          }}
+          <h4 class='p-heading-icon__title'>Case study</h4>
+        </div>
+      </div>
+      <p>Ubuntu Core hardens Rigado&rsquo;s IoT gateways with cryptography-enhanced capabilities like secure boot and full disk encryption.</p>
+      <p><a href='https://ubuntu.com/engage/case-study-rigado' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'product page', 'eventAction' : 'core - left', 'eventLabel' : 'Case study - IMS Evolve ', 'eventValue' : '1' });'>Rigado IoT gateway deploys with Ubuntu&nbsp;&rsaquo;</a></p>
+      <!-- rtp section right end -->
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <!-- Connected OT -->
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h2>Connected OT</h2>
+      <p class="p-heading--four">Harness data to supercharge shop floor productivity</p>
+      <ul>
+        <li>Build smart industrial gateways with open source</li>
+        <li>Connect your factory to the public clouds</li>
+        <li>Deploy industrial apps on your machines</li>
+        <li>Implement bespoke device fleet management</li>
+        <li>App stores including OPC-UA, Modbus, MQTT clients and more</li>
+      </ul>
+      <ul class="p-inline-images u-no-margin--bottom u-align--left">
```

Comment:
I think `p-inline-images--small` matches the design slightly closer and looks better.